src/hb-ot-cmap-table.hh

```diff
@@ -344,7 +344,17 @@ struct CmapSubtableFormat4
       {
 	unsigned int rangeOffset = this->idRangeOffset[i];
 	if (rangeOffset == 0)
-	  out->add_range (this->startCount[i], this->endCount[i]);
+	{
+	  for (hb_codepoint_t codepoint = this->startCount[i];
+	       codepoint <= this->endCount[i];
```

Comment:
Nicer if you also save endCount[i] in a variable.

Reply:
I just copied the loop from the branch below it.